nautobot/extras/factory.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.contenttypes.models import ContentType
+from django.utils.text import slugify
+
+import factory
+import faker
+
+from nautobot.core.factory import OrganizationalModelFactory
+from nautobot.extras.models import Status, Tag
+from nautobot.extras.utils import FeatureQuery, TaggableClassesQuery
+from nautobot.utilities.choices import ColorChoices
+from nautobot.utilities.factory import get_random_instances
+
+
+# Status isn't technically an OrganizationalModel, but it has all of its features **except** dynamic-groups
+class StatusFactory(OrganizationalModelFactory):
+    class Meta:
+        model = Status
+        exclude = ("has_description",)
+
+    name = factory.LazyFunction(
+        lambda: "".join(word.title() for word in faker.Faker().words(nb=2, part_of_speech="adjective", unique=True))
+    )
```

Comment:
This pattern for joining words is cleaner than the one using inner `Params` class IMO. Compare to what I did for `PlatformFactory`:

```python
    class Params:
        first = UniqueFaker("word")
        last = UniqueFaker("word")

    # Slug isn't defined here since it will always inherit from name.
    name = factory.LazyAttribute(lambda o: f"{o.first} {o.last}".title())
```

Reply:
Sorry, I'm having a bit of trouble parsing this. Are you saying you'd prefer for me to use the `Params` approach, or are you saying this is fine as-is?

Reply:
Oh sorry, I was saying I find this approach to be better/cleaner than what I was attempting. Sorry about the confusion.